Web-Application-test-case/Server/views/delay_down.ejs

```diff
@@ -24,10 +24,6 @@
                             'text': "click"
                             }).hide().appendTo("body")[0].click();
                             i = i + 1;
-
-                            if(i>=6){
-                              clearInterval(intervalId);
-                            }
                         }, 10000);
```

Comment:
Want to try rewriting this function using vanilla JS so we can get rid of JQuery?